components/Dialogs/src/private/MDCAlertControllerView+Private.m

```diff
@@ -230,6 +230,17 @@ - (void)updateButtonFont {
   }
   for (MDCFlatButton *button in self.actionButtons) {
     [button setTitleFont:finalButtonFont forState:UIControlStateNormal];
+    [button sizeToFit];
+    CGRect buttonRect = button.frame;
+    buttonRect.size.height = MAX(CGRectGetHeight(buttonRect), MDCDialogActionButtonHeight);
+    buttonRect.size.width = MAX(CGRectGetWidth(buttonRect), MDCDialogActionButtonMinimumWidth);
+    button.frame = buttonRect;
+    CGFloat verticalInset = MIN(0, -(48 - CGRectGetHeight(button.bounds)) / 2);
+    CGFloat horizontalInset = MIN(0, -(48 - CGRectGetWidth(button.bounds)) / 2);
```

Comment:
We can open an issue to deal with later, but 48 should likely be declared somewhere as something like MINIMUM_TOUCH_TARGET so 48 won't be magically scattered around our code.